di-ipv-core-stub/deploy/cri/authorizer/index.js

```diff
@@ -0,0 +1,39 @@
+const AWS = require('aws-sdk');
+AWS.config.update({region: 'eu-west-2'});
+
+export const handler = async(event) => {
+    let response = {
+        "isAuthorized": false,
+        "context": {
+            "Authorization": "failed"
+        }
+    };
+
+    const auth = await getParam();
+    var authorizationHeader = event.headers.Authorization
```

Comment:
Not a biggie but it's best practice to use let/const not var as var has some weird scoping behaviour